Src/Particle/AMReX_ParticleContainerI.H

```diff
@@ -646,6 +646,36 @@ ParticleContainer<NStructReal, NStructInt, NArrayReal, NArrayInt, Allocator>::Re
   }
 }
 
+struct TransformerVirt
+{
+    /**
+       \brief This copies the particle but replaces the id with VirtualParticleID
+       *
+     **/
+    TransformerVirt ()
```

Comment:
I think you can remove this and let the compiler generate this for you.

Reply:
Pushed a commit which removes this